Makefile

```diff
@@ -127,13 +128,35 @@ docker-push:
 	@echo "Pushing docker image: $(DOCKER_TAG)"
 	@-docker push "$(DOCKER_TAG)"
 
+.PHONY: docker-stop
+docker-stop:
+	@echo "Stopping test docker image: $(DOCKER_TEST)"
+	@-docker container stop "$(shell docker container ls -q --filter name=$(DOCKER_TEST))" 2>/dev/null || true
```

Comment:
Just curious, what's the different between using directly the name DOCKER_TEST versus doing `docker container ls -q --filter name=$(DOCKER_TEST)`?

Reply:
You're right! We can simplify that.